2/3_dial_bootstrap.md

```diff
@@ -1,13 +1,27 @@
-* You can dial to a multiadr using libp2p .dial
-Try it yourlsef
-* init multiadr, dail libp2p (some more description about the apis)
+Connecting to a bootstrap node
+==============================
 
-Try it locally
-* open one terminal and run the bootsrap 
-* and from other term run index, you would see the connect log!
+Now that we understand the concept of a bootnode, let's think about how to connect to one from another node.  To do that, we have to learn about how to create connections between libp2p nodes.  In libp2p, there are two ways to create connections.  You either use the `dial` function to create a generic connection with another node or the `dialProtocol` function to create a connection using a specific protocol (e.g. `/chat/1.0.0`).  For now, we're just going to worry about connecting to other peers so let's use the bootstrap node we created in the last section as a remote peer and dial a connection.  
 
-Now you are able to make a connection between the bootstrap and a user. This concludes the chapter. In the next chapter we would try to send and recieve data through the establised connection.
+The [`dial`](https://github.com/libp2p/js-libp2p/blob/master/doc/API.md#dial) function takes either a `PeerId` or a multiaddress as a parameter and will attempt to create a connection to that peer, something like below.
+```javascript
+const conn = await libp2p.dial(bootstrapNodeMultiaddr)
+```
 
+If the connection is created successfully, both nodes will then emit a `peer:connect` event.
+
+We'll get more into what you do with the [`connection`](https://github.com/libp2p/js-libp2p-interfaces/tree/master/src/connection) object in a future section.
+
+Try it yourself
+===============
+* Start up the bootstrap node created in the last section
+* Copy the node file and then change the port numbers in the `options` object to create a new node
+* Add code to dial the bootstrap node from the second node and see what happens
+* Run the nodes
+ - Open one terminal and run the bootstrap 
+ - Open a second terminal and run your second node. You should see a connection message since both nodes have a listener configured to listen for the `peer:connection` event.
```

Comment:
```suggestion
===============
* In `index.js` add the code to dial the bootstrap node
* Run the `bootstrap.js` 
* From another terminal run `index.js`
* You should see a connection message since both nodes have a listener configured to listen for the `peer:connection` event.
```